HomeWork 2 THE LAAAASTLAAAST VERSION

l0gark opened the pull request:
Логинов

l0gark committed:
HomeWork 2 THE LAAAASTLAAAST VERSION

Логинов

tariel-x commented on main.go:
```diff
@@ -1,70 +1,67 @@
-	names := os.Args
+func initFiles() {
+	var names []string
+	names = append(names, "aa.txt", "bb.txt", "cc.txt")
```
Это вместо входных параметров?

tariel-x commented on main.go:
```diff
@@ -1,70 +1,67 @@
+	"bufio"
 	"fmt"
 	"inverseIndex2/myFile"
+	"inverseIndex2/myIndex"
```
`"github.com/polis-mail-ru...."`

tariel-x commented:
Github пишет, что существует конфликт между веткой master и текущей.

l0gark commented:
Я эти файлы использую, для того чтоб через консоль не запускать во время отладки, поэтому я их закомментил


Отправлено из приложения "Почта" для Windows 10

От: Nikita Gerasimov
Отправлено: 22 октября 2018 г. в 20:28
Кому: polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy
Копия: LoginovArkadiy; Author
Тема: Re: [polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy]HomeWork 2 THE LAAAASTLAAAST VERSION (#6)

@tariel-x commented on this pull request.

In main.go:
>  

-	fmt.Println("Hello World!")

-	var files []myFile.MyFile

-	//var names []string

-	//names = append(names, "aa.txt", "bb.txt", "cc.txt")

-	names := os.Args

+func initFiles() {

+	var names []string

+	names = append(names, "aa.txt", "bb.txt", "cc.txt")

Это вместо входных параметров?

In main.go:
>  	"fmt"

 	"inverseIndex2/myFile"

+	"inverseIndex2/myIndex"

"github.com/polis-mail-ru...."
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

l0gark commented:
Видимо я где то ошибся когда отправлял, я попытался закоммитить в существующую ветку, как вы говорили, но помоему безуспешно, возможно здесь косяк

Отправлено из приложения "Почта" для Windows 10

От: Nikita Gerasimov
Отправлено: 22 октября 2018 г. в 20:28
Кому: polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy
Копия: LoginovArkadiy; Author
Тема: Re: [polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy]HomeWork 2 THE LAAAASTLAAAST VERSION (#6)

Github пишет, что существует конфликт между веткой master и текущей.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

tariel-x commented:
> Я эти файлы использую, для того чтоб через консоль не запускать во время отладки, поэтому я их закомментил Отправлено из приложения "Почта" для Windows 10 От: Nikita Gerasimov Отправлено: 22 октября 2018 г. в 20:28 Кому: polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy Копия: LoginovArkadiy; Author Тема: Re: [polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy]HomeWork 2 THE LAAAASTLAAAST VERSION (#6) @tariel-x commented on this pull request. In main.go:
> -	fmt.Println("Hello World!") -	var files []myFile.MyFile -	//var names []string -	//names = append(names, "aa.txt", "bb.txt", "cc.txt") -	names := os.Args +func initFiles() { +	var names []string +	names = append(names, "aa.txt", "bb.txt", "cc.txt") Это вместо входных параметров? In main.go:
> "fmt"
> "inverseIndex2/myFile" +	"inverseIndex2/myIndex" "github.com/polis-mail-ru...." — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

В командной строке можно просто нажимать стрелку вверх, чтоб повторить команду. Или можно настроить LiteIDE, чтоб она запускала с нужными параметрами.

tariel-x commented:
> Видимо я где то ошибся когда отправлял, я попытался закоммитить в существующую ветку, как вы говорили, но помоему безуспешно, возможно здесь косяк Отправлено из приложения "Почта" для Windows 10 От: Nikita Gerasimov Отправлено: 22 октября 2018 г. в 20:28 Кому: polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy Копия: LoginovArkadiy; Author Тема: Re: [polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy]HomeWork 2 THE LAAAASTLAAAST VERSION (#6) Github пишет, что существует конфликт между веткой master и текущей. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

Можно попробовать предварительно создать рабочую резервную копию файлов. Закрыть все pull request-ы, удалить все лишние ветки, удалить отдельным коммитом из master всё лишнее. Создать от master одну новую ветку, закоммитить в неё файлы из резервной копии. Тогда будет всё будет чисто и аккуратно.

l0gark commented:
Мне это переделать, вы за данный вариант оценку не поставите?


Отправлено из приложения "Почта" для Windows 10

От: Nikita Gerasimov
Отправлено: 22 октября 2018 г. в 20:45
Кому: polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy
Копия: LoginovArkadiy; Author
Тема: Re: [polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy]HomeWork 2 THE LAAAASTLAAAST VERSION (#6)

Видимо я где то ошибся когда отправлял, я попытался закоммитить в существующую ветку, как вы говорили, но помоему безуспешно, возможно здесь косяк Отправлено из приложения "Почта" для Windows 10 От: Nikita Gerasimov Отправлено: 22 октября 2018 г. в 20:28 Кому: polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy Копия: LoginovArkadiy; Author Тема: Re: [polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy]HomeWork 2 THE LAAAASTLAAAST VERSION (#6) Github пишет, что существует конфликт между веткой master и текущей. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.
Можно попробовать предварительно создать рабочую резервную копию файлов. Закрыть все pull request-ы, удалить все лишние ветки, удалить отдельным коммитом из master всё лишнее. Создать от master одну новую ветку, закоммитить в неё файлы из резервной копии. Тогда будет всё будет чисто и аккуратно.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

tariel-x commented:
> Мне это переделать, вы за данный вариант оценку не поставите? Отправлено из приложения "Почта" для Windows 10 От: Nikita Gerasimov Отправлено: 22 октября 2018 г. в 20:45 Кому: polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy Копия: LoginovArkadiy; Author Тема: Re: [polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy]HomeWork 2 THE LAAAASTLAAAST VERSION (#6) Видимо я где то ошибся когда отправлял, я попытался закоммитить в существующую ветку, как вы говорили, но помоему безуспешно, возможно здесь косяк Отправлено из приложения "Почта" для Windows 10 От: Nikita Gerasimov Отправлено: 22 октября 2018 г. в 20:28 Кому: polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy Копия: LoginovArkadiy; Author Тема: Re: [polis-mail-ru-golang-1/t2-invert-index-search-LoginovArkadiy]HomeWork 2 THE LAAAASTLAAAST VERSION (#6) Github пишет, что существует конфликт между веткой master и текущей. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread. Можно попробовать предварительно создать рабочую резервную копию файлов. Закрыть все pull request-ы, удалить все лишние ветки, удалить отдельным коммитом из master всё лишнее. Создать от master одну новую ветку, закоммитить в неё файлы из резервной копии. Тогда будет всё будет чисто и аккуратно. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

Могу в принципе, вместе с третьим заданием. Но всё равно надо, чтоб в итоге был чистый pull request без конфликтов с реализацией необходимого функционала.

l0gark committed:
Многопоточность

Добавил Горутины, посмотрите что исправить
для того чтобы остановить основной поток, я заставляю читать его из канала не знаю на сколько это правильно, когда сделал без каналов, то иногда работало иногда нет - потому что он выводил информацию до того как отработали все горутины, ну я так думаю
